1.9/usage/gpu/gpu-app-definition.md

```diff
@@ -0,0 +1,76 @@
+---
+post_title: Configure Your Service
+nav_title: Configure Your Service for GPUs
```

Comment:
The purpose of nav_title in dcos.io is generally for a shorter versions of a title, not longer.